File: app/src/main/java/com/bos/payment/appName/data/model/travel/flight/flightticketcancelResponse/ResponseHeader.java
package com.bos.payment.appName.data.model.travel.flight.flightticketcancelResponse;

import com.google.gson.annotations.SerializedName;

public class ResponseHeader{

	@SerializedName("status_Id")
	private String statusId;

	@SerializedName("error_Code")
	private String errorCode;

	@SerializedName("error_Desc")
	private String errorDesc;

	@SerializedName("error_InnerException")
	private String errorInnerException;

	@SerializedName("request_Id")
	private String requestId;

	public String getStatusId(){
		return statusId;
	}

	public String getErrorCode(){
		return errorCode;
	}

	public String getErrorDesc(){
		return errorDesc;
	}

	public String getErrorInnerException(){
		return errorInnerException;
	}

	public String getRequestId(){
		return requestId;
	}
}

File: app/src/main/java/com/bos/payment/appName/data/model/travel/flight/airbookingticketList/ResponseHeader.kt
package com.bos.payment.appName.data.model.travel.flight.airbookingticketList

import com.google.gson.annotations.SerializedName

data class ResponseHeader(@SerializedName("status_Id")
                          val statusId: String = "",
                          @SerializedName("error_Code")
                          val errorCode: String = "",
                          @SerializedName("error_Desc")
                          val errorDesc: String = "",
                          @SerializedName("error_InnerException")
                          val errorInnerException: String = "",
                          @SerializedName("request_Id")
                          val requestId: String = "")